bin/infrasim-init

```diff
@@ -10,8 +13,6 @@ import netifaces
 
 INFRASIM_TEMP_CONF = "/usr/local/etc/infrasim/conf/infrasim.yml"
 INFRASIM_CONF = "/etc/infrasim/infrasim.yml"
-VBMC_TEMP_CONF = "/usr/local/etc/infrasim/conf/vbmc.conf"
-VBMC_CONF = "/etc/infrasim/vbmc.conf"
 
 mac_base = "00:60:16:"
```

Comment:
@xiar, this is EMC's MAC prefix only, so...it's better to maintain other vendor's prefix and pick up according to the runtime simulation